CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -22,6 +22,79 @@ public class ReplicatorConfigurationRequestHandler {
         /////////////////////////////
             
         // TODO: Change client to expect replicator config, not the builder.
+        case "replicatorCollection_configure":
+            let conflictResolver: ConflictResolverProtocol? = args.get(name: "conflictResolver")
+            let pull_filter: Bool? = args.get(name: "pull_filter")!
+            let push_filter: Bool? = args.get(name: "push_filter")!
+            let filter_callback_func: String? = args.get(name: "filter_callback_func")
+            let channels: [String]? = args.get(name: "channels")
+            let documentIDs: [String]? = args.get(name: "documentIDs")
+            var config = CollectionConfiguration()
+            if(conflictResolver != nil) {
+                config.conflictResolver = conflictResolver
+            }
+            if pull_filter != false {
+                if filter_callback_func == "boolean" {
+                    config.pullFilter = _replicatorBooleanFilterCallback;
+                } else if filter_callback_func == "deleted" {
+                    config.pullFilter = _replicatorDeletedFilterCallback;
+                } else if filter_callback_func == "access_revoked" {
+                    config.pullFilter = _replicatorAccessRevokedCallback;
+                } else {
+                    config.pullFilter = _defaultReplicatorFilterCallback;
+                }
+            }
+            if push_filter != false {
+                if filter_callback_func == "boolean" {
+                    config.pushFilter = _replicatorBooleanFilterCallback;
+                } else if filter_callback_func == "deleted" {
+                    config.pushFilter = _replicatorDeletedFilterCallback;
+                } else if filter_callback_func == "access_revoked" {
+                    config.pushFilter = _replicatorAccessRevokedCallback;
+                } else {
+                    config.pushFilter = _defaultReplicatorFilterCallback;
+                }
+            }
+            if(channels != nil) {
+                config.channels = channels
+            }
+            if(documentIDs != nil){
+                config.documentIDs = documentIDs
+            }
```

Comment:
same as above, set it without `if condition`

Reply:
here documnetIDs is an array. In xcode it is giving error without nil. 
Optional type '[String]?' cannot be used as a boolean; test for '!= nil' instead

Reply:
I think the documentIDs is `[String]?` 
https://github.com/couchbase/couchbase-lite-ios/blob/master/Swift/ReplicatorConfiguration.swift#L123

I mean to do this 
```
config.documentIDs = documentIDs 
```

instead of 
```
            if(documentIDs != nil){
                config.documentIDs = documentIDs
            }
```